packages/cli/src/commands/test.ts

```diff
@@ -179,7 +179,10 @@ export default class Test extends AuthCommand {
       })
       process.exitCode = 1
     })
-    runner.on(Events.RUN_FINISHED, () => reporter.onEnd())
+    runner.on(Events.RUN_FINISHED, () => {
+      reporter.onEnd()
+      this.exit()
```

Comment:
I don't think that we should need this. `await runner.run()` on the next line should already wait until all of the checks are finished. Is there a bug where `await runner.run()` doesn't actually terminate when all of the checks are finished? If so, then I think that we should address that.